nss-day-cohort-41/tabloidcli-the-beaker-bunsen-brunch-club-mo
# Author Crud and welcome message

# Description
- confirmed full crud functionality for authors
- added public method ChooseAuthor() to make the Choose() method publicly accessable
- added initial greeting

Fixes # (issue)
## Type of change

# Testing Instructions
- Upon cloning the repo navigate to the home directory 
- dotnet run
- greeting should populate upon start up
- select the author option and a menu will be presented with the various CRUD options
# Checklist:
